docs/workspaces.md

```diff
@@ -534,7 +534,7 @@ workspaces:
 
 ##### `csi`
 
-This is an alpha feature. The `enable-api-fields` feature flag [must be set to `"alpha"`](./install.md)
+This is a beta feature. The `enable-api-fields` feature flag [must be set to `"beta"`](./install.md)
```

Comment:
So, I think we only need to set it to "beta" for the v1 API version, not the v1beta1 api version